src/Features/CSharp/Portable/CodeRefactorings/ConvertLocalFunctionToMethod/CSharpConvertLocalFunctionToMethodCodeRefactoringProvider.cs

```diff
@@ -42,23 +42,30 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var cancellationToken = context.CancellationToken;
             var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
-            var identifier = await root.SyntaxTree.GetTouchingTokenAsync(context.Span.Start,
-                token => token.Parent.IsKind(SyntaxKind.LocalFunctionStatement), cancellationToken).ConfigureAwait(false);
-            if (identifier == default)
+            LocalFunctionStatementSyntax localFunction = default;
+            if (context.Span.IsEmpty)
             {
-                return;
+                // This branch is required to handle "C LocalFunction[||](C c)" -> root.FindNode return ParameterList
+                var identifier = await root.SyntaxTree.GetTouchingTokenAsync(context.Span.Start, token => token.Parent.IsKind(SyntaxKind.LocalFunctionStatement), cancellationToken).ConfigureAwait(false);
```

Comment:
We have `GetTouchingWord` which prefers identifiers over punctuation.

Reply:
Since I'm testing on `LocalFunctionStatement` it doesn't make any difference and would only create one additional check (first on IsWord and then on is LocalFunctionStatement)

Reply:
> Since I'm testing on LocalFunctionStatement it doesn't make any difference and would only create one additional check (first on IsWord and then on is LocalFunctionStatement)

I guess i'm just saying: GetTouchingWord is just the easier thing to use.  But up to you.